comploinc_config.yaml

```diff
@@ -41,13 +45,13 @@ loinc_snomed:
         description: 'loinc_snomed_release/20231015/Snapshot/Terminology/xsct2_Description_Snapshot-en_LO1010000_20231015.txt'
         identifier: 'loinc_snomed_release/20231015/Snapshot/Terminology/xsct2_Identifier_Snapshot_LO1010000_20231015.txt'
         relationship: 'loinc_snomed_release/20231015/Snapshot/Terminology/xsct2_Relationship_Snapshot_LO1010000_20231015.txt'
-        part_mapping: 'loinc_snomed_release/part-mappings_0.0.3.tsv'
+        part_mapping: 'loinc_snomed_release/20231015/part-mappings_0.0.3.tsv'
```

Comment:
Probably best to keep these versioned differently, since they don't come with the actual LOINC-SNOMED release. Perhaps `loinc_snomed_release/part_mappings/`? And can then have name variations for them? Like end with YYYY-MM-DD, or otherwise just keep as is with semver.